plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1187,7 +1185,7 @@ elementgui.tool.usage_count=Number of uses / durability:
 elementgui.tool.blocks_affected=Blocks affected: 
 elementgui.tool.stays_in_grid_when_crafting=Does item stay in crafting grid when crafted?
 elementgui.tool.damaged_on_crafting=<html>Damage item instead on crafting<br><small>Make sure to enable "stay in crafting grid" and that item is damageable
-elementgui.tool.tooltip_tip=<html>Special information about the tool:<br><small>Separate entries with comma, to use comma in description use \,
+elementgui.tool.tooltip_tip=<html>Special information about the tool:<br><small>Separate entries with comma, to use comma in description use,

```

Comment:
You need to place 2 `\\` at the end of the line :
`elementgui.tool.tooltip_tip=<html>Special information about the tool:<br><small>Separate entries with comma, to use comma in description use\\,`